Mathlib/GroupTheory/FreeGroup/Reduce.lean

```diff
@@ -214,6 +214,17 @@ theorem reduce_toWord : ∀ x : FreeGroup α, reduce (toWord x) = toWord x := by
 theorem toWord_one : (1 : FreeGroup α).toWord = [] :=
   rfl
 
+@[to_additive]
+theorem toWord_mul (x y : FreeGroup α) : toWord (x * y) = reduce (toWord x ++ toWord y) := by
+  rw [← mk_toWord (x := x), ← mk_toWord (x:= y), mul_mk]
+  simp
+
+@[to_additive]
+theorem toWord_pow (x : FreeGroup α) (n : ℕ) :
+    toWord (x^n) = reduce (List.replicate n x.toWord).flatten := by
+  rw [← mk_toWord (x := x), pow_mk]
```

Comment:
It's a minor nit, but I figure since this compiles without this lemma in the simp afterwards, better to leave it out
```suggestion
  rw [← mk_toWord (x := x)]
```